EC restores incorrect `time.Time` value

<!-- Provide a general summary of the issue in the Title above -->

## Expected Behavior
<!-- If you're describing a bug, tell us what should happen
     If you're suggesting a change/improvement, tell us how it should work -->
In the new session token v2, `time.Time` was used for the token's lifetime. For the restored object, it is expected that the token will be restored in the same way.

## Current Behavior
<!-- If describing a bug, tell us what happens instead of the expected behavior
     If suggesting a change/improvement, explain the difference from current behavior -->
The lifetime field of the restored object token differs from the original one:
```
=== RUN   Test_Slicing_REP3/no_payload
=== RUN   Test_Slicing_REP3/no_payload/sessionv2
    service_test.go:536: 
        	Error Trace:	/neofs-node/pkg/services/object/put/service_test.go:536
        	            				/neofs-node/pkg/services/object/put/service_test.go:551
        	            				/neofs-node/pkg/services/object/put/service_test.go:366
        	Error:      	Not equal: 
        	               ...
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -3,4 +3,4 @@
        	            	   iat: (time.Time) {
        	            	-   wall: (uint64) 14002080705023339520,
        	            	-   ext: (int64) 18550458,
        	            	+   wall: (uint64) 0,
        	            	+   ext: (int64) 63903828060,
        	            	    loc: (*time.Location)({
        	            	@@ -575,4 +575,4 @@
        	            	   nbf: (time.Time) {
        	            	-   wall: (uint64) 14002080705023339520,
        	            	-   ext: (int64) 18550458,
        	            	+   wall: (uint64) 0,
        	            	+   ext: (int64) 63903828060,
        	            	    loc: (*time.Location)({
        	            	@@ -1147,4 +1147,4 @@
        	            	   exp: (time.Time) {
        	            	-   wall: (uint64) 14002119359729003520,
        	            	-   ext: (int64) 36000018550458,
        	            	+   wall: (uint64) 0,
        	            	+   ext: (int64) 63903864060,
        	            	    loc: (*time.Location)({
        	Test:       	Test_Slicing_REP3/no_payload/sessionv2
--- FAIL: Test_Slicing_REP3/no_payload/sessionv2 (0.00s)
```

## Steps to Reproduce (for bugs)
<!-- Provide a link to a live example, or an unambiguous set of steps
     to reproduce this bug. -->

Use #3671. Tests by path `pkg/services/object/put/service_test.go`, uncomment the lines under `// TODO: fix Lifetime fields different` and run.

## Context
<!-- How has this issue affected you? What are you trying to accomplish?
     Providing context helps us come up with a solution that is most useful in the real world -->
When testing new session tokens v2 #3671, tests with EC fail for `time.Time` fields.

## Regression
<!-- Is this issue a regression? (Yes / No)
     If Yes, optionally please include version or commit id or PR# that caused this regression,
     if you have these details -->
Yes. EC is incompatible with the new v2 session tokens.

## Your Environment
<!-- Include as many relevant details about the environment you experienced the bug in -->
* Version used: #3671.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -51,7 +51,7 @@ Changelog for NeoFS Node
 
 ### Updated
 - `github.com/nspcc-dev/neofs-contract` module to `v0.25.2-0.20251223162726-c0cf83ca5e42` (#3670, #3746, #3733)
-- `github.com/nspcc-dev/neofs-sdk-go` module to `v1.0.0-rc.16.0.20251224112927-a50d7e9c925a` (#3711, #3750, #3733)
+- `github.com/nspcc-dev/neofs-sdk-go` module to `v1.0.0-rc.16.0.20260114103127-15d6922be29e` (#3711, #3750, #3733, #3775)
 - `github.com/nspcc-dev/locode-db` module to `v0.8.2` (#3729)
 - `github.com/nspcc-dev/neo-go` module to `v0.115.0` (#3733, #3769)
 

--- a/go.mod
+++ b/go.mod
@@ -22,7 +22,7 @@ require (
 	github.com/nspcc-dev/neo-go v0.115.0
 	github.com/nspcc-dev/neofs-api-go/v2 v2.14.1-0.20240827150555-5ce597aa14ea
 	github.com/nspcc-dev/neofs-contract v0.25.2-0.20251223162726-c0cf83ca5e42
-	github.com/nspcc-dev/neofs-sdk-go v1.0.0-rc.16.0.20251224112927-a50d7e9c925a
+	github.com/nspcc-dev/neofs-sdk-go v1.0.0-rc.16.0.20260114103127-15d6922be29e
 	github.com/nspcc-dev/tzhash v1.8.3
 	github.com/panjf2000/ants/v2 v2.11.3
 	github.com/prometheus/client_golang v1.23.2

